mac/macdeploy.sh

```diff
@@ -7,6 +7,7 @@
 
 MACDEPLOYQT="macdeployqt"
 LIBRARY_NAME="libQupZilla.2.dylib"
+PLUGINS="QupZilla.app/Contents/Resources/plugins"
```

Comment:
@nowrep 

<img src="https://cloud.githubusercontent.com/assets/114709/22442425/9856f788-e6f8-11e6-9881-e09eaf78b49c.png" width="160" height="90" title="2184 Outline" />

Okay here's what was tried last week and one tid bit today. Please refer to the image attached for the item numbers.

1. Qt source path for `libqsvgicon.dylib` is at `.../clang_64/plugins/iconengines/`
2. Target goal *where it should be* but *is not*... for this illustration it was created manually there but also programmatically with `macdeploy.sh`.  This does **not** work here even with `install_name_tool`.
3. Todays retest... it does work here and committed with fe9e41788dd08df673b48627ade7c5fac5b0d16b

A. The issue itself... when the dylib plugin is placed in a path where it is known to be loaded with `macdeployqt.sh` then it does finally load the "New Tab" and "New Window" svg icons.

I've also left the non-highlighted areas visible to show how Qt is laid out structure wise both pre-deploy and post-deploy of QupZilla.